.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: c
-sudo: required
-service: docker
+services: docker
+os: linux
+dist: xenial
```

Comment:
why must we pin the linux distro?

Reply:
this distribution is unrelated to the container used to test the code.

Travis by default uses ubuntu xenial, we just make it explicit, this protects against possible breakages when the default changes. I also don't think can be centralized, unlike the environment variables.

Reply:
confusingly we have `DISTRO="debian-9-ocaml-4.07"` below

i assume `DISTRO` corresponds to the docker distro, but `distro` corresponds to the os distro